core/cmd/app.go

```diff
@@ -147,6 +147,23 @@ func NewApp(client *Client) *cli.App {
 			Usage:  "Removes a specific bridge",
 			Action: client.RemoveBridge,
 		},
+		{
+			Name:    "externalinitiators",
+			Aliases: []string{"exi"},
+			Usage:   "Tasks for managing external initiators",
+			Subcommands: []cli.Command{
+				{
+					Name:   "create",
+					Usage:  "Create an authentication key for a user of external initiators",
```

Comment:
Would it be too much to add a `exi list` command that prints out the existing access keys? I see that being very useful for Neil.

Reply:
https://www.pivotaltracker.com/story/show/165195702